packages/client/hmi-client/src/model-representation/petrinet/petrinet-renderer.ts

```diff
@@ -9,16 +10,13 @@ export interface NodeData {
 	expression?: string;
 	strataType?: string;
 	isStratified?: boolean;
+	value?: any;
```

Comment:
Are you using this `value?: any` you introduced? Seems to be no?

Reply:
I am using it in the `mode-mouse-enter` here. Where I extract the references. I'll adjust that, there doesn't seem to be a need for an entire value property

Reply:
should be more clear now